keras_cv/layers/preprocessing/random_translation.py

```diff
@@ -116,6 +117,9 @@ class RandomTranslation(BaseImageAugmentationLayer):
       seed: Integer. Used to create a random seed.
       fill_value: a float represents the value to be filled outside the
           boundaries when `fill_mode="constant"`.
+      bounding_box_format: The format of bounding boxes of input dataset. Refer to
+          https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
+          for more details on supported bounding box formats.
```

Comment:
Let's add "Required iff augmenting data which includes bounding boxes." (This isn't currently included in other docstrings for KPLs but I think it probably should be)